plugins/modules/vmware_guest_vgpu.py

```diff
@@ -0,0 +1,362 @@
+#!/usr/bin/python
```

Comment:
Add an entry for this module here - https://github.com/ansible-collections/community.vmware/blob/44ac140e30ff0212470b47a29b105d0cdede4df4/meta/runtime.yml#L76

Reply:
Run `tox -e add_docs` to generate this module's documentation in `docs/` directory and add that as well.